lib/storage.js

```diff
@@ -666,3 +668,12 @@ function _assert128BitId(id) {
       'array of 16 random bytes.');
   }
 }
+
+function _validateDocSequence(sequence) {
```

Comment:
Might be good to also move the "seq < 0" check here to avoid repeating it twice above.

Reply:
@davidlehn Done, the non-negative sequence check is now in the private helper function.